### PR TITLE
[ci][chore] Update/tag respective stable version docs on tag

### PR DESCRIPTION
**Pull request by TheCatLady**
# Description

See someengineering/resoto.com#453, which adds multiple stable versioned docs support in preparation for the 3.0 release.

Also adds step to update data models.

# Code of Conduct

By submitting this pull request, I agree to follow the [code of conduct](https://resoto.com/code-of-conduct).

### CODE REVIEW

**Review by lloesche — DISMISSED** [review]
lgtm 🚢 🥳 

Just needs the model-wait after the compose up before merging.

File: .github/workflows/docker-build.yml [hunk 1]
```diff
@@ -274,6 +274,15 @@ jobs:
...
           token: ${{ secrets.SOME_CI_PAT }}
 
+      - name: Update resource data models
+        continue-on-error: true
+        shell: bash
+        run: |
+          PSK= RESOTOCORE_ANALYTICS_OPT_OUT=true docker-compose up -d
+          cd resoto.com/docs/reference/data-models
+          find . -mindepth 2 -type d -name img -prune -exec rm -rf {} \;
+          python ${{ github.workspace }}/resoto.com/tools/export_model_images.py
```
Comment (aquamatthias) [inline .github/workflows/docker-build.yml:284]: This will create a couple of hundred file changes for every commit, or?
Can we do better?

**Review by aquamatthias — COMMENTED** [review]
One question - otherwise lgtm. Thanks!

**Review by lloesche — APPROVED** [review]
Thank you! 🚢

**Pull request merged**